packages/theme/src/sass/tokens.scss

```diff
@@ -29,6 +29,8 @@ $tokens: (
       // Neutral - State
       'disabled': null,
       'on-disabled': null,
+      // Accent      
+      'accent-focused': null,
```

Comment:
We don't have such token in the Figma. Where does it come from?

Reply:
It is color from figma, for DEBUG status
Should I add this color locally, in component Badge.vue, instead?

Reply:
Hmm, I see. I think it is a bug of the design in Figma. I've reported it.